### DISCUSSION TITLE
ORF RAW Files from new OM-1 from olympus are not shown in View

### DISCUSSION THREAD

**Discussion by dietrest** [outdoorbits/little-backup-box#225] [Q&A]
I own Olympus OM-1 OMD-EM1, Mark II and EPL-8. The ORF Raw files from OM-1 are not shown in View from Little Backup Box but the ones from  OMD-EM1 and EPL-8 are shown. I suppose libraw-bin and libraw20 has to be newer. If I try to install from bookworm repo the box wants to upgrade a lot. for example libc6. 
Is there an Image of Bookworm available? Can I install Little Backup Box on Bookworm?

**Comment by outdoorbits** [comment]
I don't know if it's running on a bookworm image and I can't test now because I'm traveling until 20. of August. But have you tried to do all the recommended updates? Or did you try LBB on bookworm? Please send your experience! I will put this on my to-do list also.
